pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -768,12 +765,12 @@ func buildContainerMapFromRuntime(runtimeService internalapi.RuntimeService) (co
 	containerList, _ := runtimeService.ListContainers(nil)
 	for _, c := range containerList {
 		if _, exists := podSandboxMap[c.PodSandboxId]; !exists {
-			return nil, fmt.Errorf("no PodsandBox found with Id '%s'", c.PodSandboxId)
+			klog.Warningf("no PodsandBox found with PodSandboxId: %s, containerName: %s, containerId: %s", c.PodSandboxId, c.Metadata.Name, c.Id)
```

Comment:
Sorry but I think instead of removing the error you should treat the error in kuberuntime_manager

Reply:
Thank you for your suggestion, I have added err back, and recorded the error log in the containerManager.

Reply:
Ok, I apologize for the bikeshedding but I think you were right at the first place... so we warn here and maybe indicate this container has been skipped.